### PR TITLE
PMM-9378 refactor protobuf files

### PR DESCRIPTION
**Pull request by PavelKhripkov**
https://jira.percona.com/browse/PMM-9378

https://github.com/Percona-Lab/pmm-submodules/pull/2328

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

File: api/serverpb/server.proto [hunk 1]
```diff
@@ -291,61 +323,97 @@ service Server {
...
   }
   // TestEmailAlertingSettings tests email alerting SMTP settings by sending testing email.
   rpc TestEmailAlertingSettings(TestEmailAlertingSettingsRequest) returns (TestEmailAlertingSettingsResponse) {
     option (google.api.http) = {
       post: "/v1/Settings/TestEmailAlertingSettings"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "TestEmailAlertingSettings"
+      description: "Tests email alerting SMTP settings by sending testing email."
```
Comment (catalinaadam) [inline api/serverpb/server.proto:405]: Sends test email to check current SMTP settings for email alerting.

**Review by catalinaadam — COMMENTED** [review]
Suggestions for improving readibility

File: api/serverpb/server.proto [hunk 2]
```diff
@@ -291,61 +323,97 @@ service Server {
...
-  // Readiness returns an error when some PMM Server component is not ready yet or is being restarted.
-  // It can be used as for Docker health check or Kubernetes readiness probe.
+  // Readiness returns an error when Server components being restarted are not ready yet.
+  // Use this API for checking the health of Docker containers and for probing Kubernetes readiness.
   rpc Readiness(ReadinessRequest) returns (ReadinessResponse) {
     option (google.api.http) = {
       get: "/v1/readyz"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Readiness"
```
Comment (atymchuk) [inline api/serverpb/server.proto:338]:
```suggestion
      summary: "Check server readiness"
```

**Review by atymchuk — CHANGES_REQUESTED** [review]
Good job! Just a minor correction please )

**Review by atymchuk — DISMISSED** [review]

**Review by catalinaadam — APPROVED** [review]

**Review by catalinaadam — APPROVED** [review]

**Pull request merged**